lib/ex_aws/config/defaults.ex

```diff
@@ -17,7 +17,7 @@ defmodule ExAws.Config.Defaults do
       region: "us-east-1",
       port: 80
     },
-    kinesis_firehose: %{
+    firehose: %{
```

Comment:
Actually I had to change that in order to get it working. Looks like the service it's just called  firehose on amazon. 
I can change the module to `Firehose` if you prefer that.